source/apiVolontaria/apiVolontaria/factories.py

```diff
@@ -34,17 +34,17 @@ class Meta:
 
     first_name = factory.Faker('first_name')
     last_name = factory.Faker('last_name')
-    username = factory.Sequence('Chuck{0}'.format)
-    email = factory.Sequence('chuck{0}@volontaria.com'.format)
+    username = factory.Faker('email')
+    email = factory.Faker('email')
     password = 'Test123!'
     is_superuser = True
 
     @classmethod
-    def _prepare(cls, create, **kwargs):
+    def _create(cls, model_class, *args, **kwargs):
         if 'password' in kwargs.keys():
             password = kwargs.pop('password', None)
 
-        user = super(AdminFactory, cls)._prepare(create, **kwargs)
+        user = super(AdminFactory, cls)._create(model_class, *args, **kwargs)
 
         if password:
             user.set_password(password)
```

Comment:
The _prepare function does not exist in any of the bases classes.  I have change it to _create.

Reply:
Good move :+1: 
I double check on the documentation and source code and you're right.